guides/md/keras_cv/generate_images_with_stable_diffusion.md

```diff
@@ -0,0 +1,513 @@
+    "An oldschool macintosh computer showing an avocado on its screen",
+)
+plot_images(images)
+```
+
+<div class="k-default-codeblock">
+```
+25/25 [==============================] - 31s 154ms/step
+
+```
+</div>
+    
+![png](/img/guides/generate_images_with_stable_diffusion/generate_images_with_stable_diffusion_20_1.png)
+    
+
+
+Let's benchmark our XLA model:
+
+
+```python
+start = time.time()
+images = model.text_to_image(
+    "A cute otter in a rainbow whirlpool holding shells, watercolor",
+    batch_size=3,
+)
+end = time.time()
+benchmark_result.append(["XLA", end - start])
+plot_images(images)
+
+print(f"With XLA: {(end - start):.2f} seconds")
+keras.backend.clear_session()
+```
+
+<div class="k-default-codeblock">
+```
+25/25 [==============================] - 39s 246ms/step
+With XLA: 56.22 seconds
+
+```
+</div>
+    
+![png](/img/guides/generate_images_with_stable_diffusion/generate_images_with_stable_diffusion_22_1.png)
+    
+
+
+On an A100 GPU, we get about a 2x speedup.  Fantastic!
+
+---
+## Putting it all together
+
+So, how do you assemble the world's most performant stable diffusion inference
+pipeline (as of September 2022).
+
+With these two lines of code:
+
+
+```python
+keras.mixed_precision.set_global_policy("mixed_float16")
+model = keras_cv.models.StableDiffusion(jit_compile=True)
+```
+
+And to use it...
+
+
+```python
+# Let's make sure to warm up the model
+images = model.text_to_image(
+    "Teddy bears conducting machine learning research",
+    batch_size=3,
+)
+plot_images(images)
+```
+
+<div class="k-default-codeblock">
+```
+25/25 [==============================] - 40s 157ms/step
+
+```
+</div>
+    
+![png](/img/guides/generate_images_with_stable_diffusion/generate_images_with_stable_diffusion_27_1.png)
+    
+
+
+Exactly how fast is it?
+Let's find out!
+
+
+```python
+start = time.time()
+images = model.text_to_image(
+    "A mysterious dark stranger visits the great pyramids of egypt, "
+    "high quality, highly detailed, elegant, sharp focus, "
+    "concept art, character concepts, digital painting",
+    batch_size=3,
+)
+end = time.time()
+benchmark_result.append(["XLA + Mixed Precision", end - start])
+plot_images(images)
+
+print(f"XLA + mixed precision: {(end - start):.2f} seconds")
+```
+
+<div class="k-default-codeblock">
+```
+25/25 [==============================] - 4s 157ms/step
+XLA + mixed precision: 4.21 seconds
+
+```
+</div>
+    
+![png](/img/guides/generate_images_with_stable_diffusion/generate_images_with_stable_diffusion_29_1.png)
+    
+
+
+Let's check out the results:
+
+
+```python
+print("{:<20} {:<20}".format("Model", "Runtime"))
+for result in benchmark_result:
+    name, runtime = result
+    print("{:<20} {:<20}".format(name, runtime))
+```
+
+<div class="k-default-codeblock">
+```
+Model                Runtime             
+Standard             8.164055824279785   
+Mixed Precision      6.0718467235565186  
+XLA                  56.22104263305664   
```

Comment:
@fchollet looks like we forgot to warm up the xla model. 

I will update and re-generate

Reply:
Done